eng/helix/content/InstallAppRuntime.ps1

```diff
@@ -0,0 +1,49 @@
+ <# 
+ .SYNOPSIS 
+     Installs an AspNetCore shared framework on a machine
+ .DESCRIPTION 
+     This script installs an AspNetCore shared framework on a machine
+ .PARAMETER AppRuntimePath
+     The path to the app runtime package to install.
+ .PARAMETER InstallDir
+     The directory to install the shared framework to.     
+ .PARAMETER Framework
+     The framework directory to copy the shared framework from.
+ #> 
+param(
+    [Parameter(Mandatory = $true)]
+    $AppRuntimePath,
+    
+    [Parameter(Mandatory = $true)]
+    $InstallDir,
+    
+    [Parameter(Mandatory = $true)]
+    $Framework)
+
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue' # Workaround PowerShell/PowerShell#2138
+
+Set-StrictMode -Version 1
+
+Write-Host "Extracting to $InstallDir"
+
+$zipPackage = [io.path]::ChangeExtension($AppRuntimePath, ".zip")
+Write-Host "Renaming to $zipPackage"
+Rename-Item -Path $AppRuntimePath -NewName $zipPackage
+if (Get-Command -Name 'Microsoft.PowerShell.Archive\Expand-Archive' -ErrorAction Ignore) {
+    # Use built-in commands where possible as they are cross-plat compatible
+    Microsoft.PowerShell.Archive\Expand-Archive -Path $zipPackage -DestinationPath ".\tmpRuntime" -Force
+}
+else {
+    Remove-Item ".\tmpRuntime" -Recurse -ErrorAction Ignore
+    # Fallback to old approach for old installations of PowerShell
+    Add-Type -AssemblyName System.IO.Compression.FileSystem
+    [System.IO.Compression.ZipFile]::ExtractToDirectory($zipPackage, ".\tmpRuntime")
+}
+
+Get-ChildItem -Path ".\tmpRuntime" -Recurse
+
+Write-Host "Copying managed files to $InstallDir"
+Copy-Item -Path ".\tmpRuntime\runtimes\win-x64\lib\$Framework\*" $InstallDir
+Write-Host "Todo: Copying native files to $InstallDir"
+# Copy-Item -Path ".\tmpRuntime\runtimes\win-x64\native\*" $InstallDir
```

Comment:
@dougbu the sharedframework bits seem to be missing the x64 native bits, do you know what part of the build these would normally come from?  They appear in the x86 version, but for some reason the native 64 bit files aren't in the AppRuntime being built here....

Reply:
I don't have a good answer for you. You could try adding `-buildNative` to the `.\build.cmd` script in the YAML if this remains a problem. Another idea would be to create artifacts containing at least the sharedFx packages you're sending to Helix agents, download them, and see if anything else is missing.